ui/components/component-library/base-icon/base-icon.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import Box from '../../ui/box/box';
+
+import { SIZES } from '../../../helpers/constants/design-system';
+
+export const BaseIcon = ({
+  className,
+  size = SIZES.MD,
+  color = 'inherit',
```

Comment:
Update to 
```suggestion
  color = COLORS.INHERIT,
```

Reply:
Udpated https://github.com/MetaMask/metamask-extension/pull/15609/commits/8b173b3ba673d1a4ef2b82e93dc4f5eb8fd4a93c